contrib/win32/openssh/sshTelemetry.c

```diff
@@ -119,6 +119,19 @@ void send_encryption_telemetry(const char* direction,
     TraceLoggingUnregister(g_hProvider1);
 }
 
+void send_exit_code_telemetry(const int exit_code)
+{
+    TraceLoggingRegister(g_hProvider1);
+    TraceLoggingWrite(
+        g_hProvider1,
+        "ExitCode",
```

Comment:
Instead of "ExitCode", maybe name it "ExitCodeForKexExchId" or something of the sort